### PR TITLE
feat: bleu 456 hero section

### PR DESCRIPTION
**Pull request by luizmuraro**
Description: header and first section of bleu's site

image

TODO: Insert right asset, create mdx structure

### CODE REVIEW

File: src/lib/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import clsx, { type ClassValue } from "clsx"
+import { twMerge } from "tailwind-merge"
+
+const cn = (...inputs: ClassValue[]) => {
```
Comment (ribeirojose) [inline src/lib/utils.tsx:4]: 👌

File: tailwind.config.js [hunk 2]
```diff
@@ -1,10 +1,59 @@
...
-    extend: {},
+    extend: {
+      colors: {
+        primary: {
+          50: "#e6edff",
+          100: "#ccdaff",
+          200: "#99b5ff",
+          300: "#6690ff",
+          400: "#336bff",
+          500: "#0043ff", // Main primary
```
Comment (ribeirojose) [inline tailwind.config.js]: `#curious` how did you generate the shades? Did find using them useful at all?

File: tailwind.config.js [hunk 3]
```diff
@@ -1,10 +1,59 @@
...
+          200: "#ffd5ef",
+          300: "#ffc1e7",
+          400: "#ffadeb", // Main secundary
+          500: "#ff99e0",
+          600: "#ff85d5",
+          700: "#ff70ca",
+          800: "#ff5cbf",
+          900: "#ff47b4"
+        },
+        neutral: {
```
Comment (ribeirojose) [inline tailwind.config.js:9]: is this the same naming we used in the figma file? Also would this work if we were using a dark theme?

File: tailwind.config.js [hunk 4]
```diff
@@ -1,10 +1,59 @@
...
+        }
+      },
+      fontFamily: {
+        sans: ["var(--font-barlow-condensed)"]
+      },
+      fontWeight: {
+        regular: 400,
+        medium: 500,
+        bold: 700
+      }
```
Comment (ribeirojose) [inline tailwind.config.js]: thought this was the default? 🤔

File: src/app/layout.tsx [hunk 5]
```diff
@@ -1,18 +1,31 @@
...
-  );
+  )
 }
 
 export const metadata = {
   title: "Create Next App",
-  description: "Generated by create next app",
-};
+  description: "Generated by create next app"
+}
```
Comment (ribeirojose) [inline src/app/layout.tsx:31]: I think the formatting is a bit off here 😬

File: public/favicon.ico
Comment (ribeirojose) [inline public/favicon.ico:1]: we could use the `B` here as favicon, wdyt?

File: public/bleu.png
Comment (ribeirojose) [inline public/bleu.png:1]: don't we have the svg here? 🤔 I think this looks slightly stretched as well

**Review by ribeirojose — APPROVED** [review]
looking good overall but would: 1) recheck that the logo is looking correct; 2) add a favicon.

**Review by ribeirojose — COMMENTED** [review]
`#would` also consider whether this color scheme would work if we were to use a dark theme

File: tailwind.config.js [hunk 6]
```diff
@@ -1,10 +1,59 @@
...
+          400: "#bdbdbd",
+          500: "#9e9e9e",
+          600: "#757575",
+          700: "#616161",
+          800: "#424242",
+          900: "#212121"
+        },
+        hover: {
+          primary: "#1a57ff",
+          secondary: "#ff94e3"
```
Comment (ribeirojose) [inline tailwind.config.js]: any reason for these not to be shades of the primary/secondary colors?

File: tailwind.config.js (hunk 2 above)
Comment (luizmuraro) [reply tailwind.config.js]: I've asked to claude generate it for me, I saw it on some projects, but thinking better we have the tailwind `/ ` for instance: bg-primary/15 that changes the opacity and have almost the same result as I wanted building this shaders, so I think that I'll remove it and keep just the primary and secondary.

File: public/favicon.ico
Comment (luizmuraro) [reply public/favicon.ico:1]: Actually, I'm already using the `B` here

File: tailwind.config.js (hunk 2 above)
Comment (ribeirojose) [reply tailwind.config.js]: yea I think Claude is probably not the right tool for the job -- if you want to keep using shades there's a couple tools you could use to generate tailwind shades (https://www.google.com/search?q=generate+tailwind+shaades&sourceid=chrome&ie=UTF-8) or Radix Colors shades https://www.radix-ui.com/colors/custom

File: public/favicon.ico
Comment (ribeirojose) [reply public/favicon.ico:1]: ops 😬 sorry about that then

File: tailwind.config.js (hunk 3 above)
Comment (luizmuraro) [reply tailwind.config.js:9]: I didn't find colors name on Figma, just the colors with his hex number.

File: tailwind.config.js (hunk 4 above)
Comment (luizmuraro) [reply tailwind.config.js]: My bad, I was testing changing these names and forget to remove it.

File: public/bleu.png
Comment (luizmuraro) [reply public/bleu.png:1]: Changed!

**Pull request merged**